### ISSUE TITLE
Any other loss function instead of crossentropy

### ISSUE THREAD

**Issue by Masrur02** [lhoyer/MIC#72]
Is it possible to use any other loss function instead of cross-entropy? I found where the cross-entropy loss function has been defined. However I did not find where it is called to calculate the loss. Can you please guide me?
